odoo/addons/base/ir/ir_cron.py

```diff
@@ -218,8 +226,21 @@ def _acquire_job(cls, db_name):
                 # we're exiting due to an exception while acquiring the lock
                 lock_cr.close()
 
+        try:
+            with db.cursor() as cr:
+                cr.execute("SELECT MIN(nextcall) as nextcall FROM ir_cron WHERE active = 't'")
+                row = cr.dictfetchone()
+                if row:
+                    res['nextcall'] = row['nextcall']
+                    min_cron = datetime.strptime(row['nextcall'], DEFAULT_SERVER_DATETIME_FORMAT)
+                    epoch = datetime.utcfromtimestamp(0)
+                    res['nextcall_epoch'] = int((min_cron - epoch).total_seconds())
+        except Exception:
+            pass
```

Comment:
Any particular reason we're silencing errors when grabbing `nextcall`? Shouldn't the caller be counting on a result that always includes a `nextcall` entry? Similarly, we might want to return `nextcall: None` if there are no future jobs planned.

Speaking of which... now that `_acquire_job()` will return something, we could add a docstring to describe what's being returned ;-)